qbft/spectest/generate/main.go

```diff
@@ -4,25 +4,33 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"log"
```

Comment:
@lior-blox or @y0sher 
I think the simple log is good enough but maybe we have reasons to use a different one?